src/main/menu/templates/view.js

```diff
@@ -113,5 +113,15 @@ export default function (keybindings) {
       }
     })
   }
+
+  viewMenu.submenu.push({
```

Comment:
You can add this directly under `Toggle Table of Contents` and then append a separator (between image reload and debug entries).